randovania/games/prime2/json_data/Torvus Bog.txt

```diff
@@ -303,6 +303,9 @@ Extra - in_dark_aether: False
                   Bomb Space Jump (Expert)
                   Movement (Intermediate) and Use Screw Attack (No Space Jump)
                   Gravity Boost and Air Underwater (Advanced)
+                  All of the following:
+                      # https://www.youtube.com/watch?v=LxH7aq404so
+                      Jump Off Enemy (Advanced) and Normal Damage ≥ 99
```

Comment:
I'd lower the damage to something like 50, so medium can still do this without etanks, but it wouldn't be in logical for lenient until you had at least 1.